shared/admin-tools/user/verify-ustc-admin-user-disabled.js

```diff
@@ -0,0 +1,7 @@
+const { verifyAdminUserDisabled } = require('./admin');
+
+(async () => {
+  console.log('== Verifying USTC admin user is disabled');
```

Comment:
Question/nit-pick: Depending on where the logs are presented, does `ApplicationContext.logger.info('== Verifying USTC admin user is disabled`)` work better?

Reply:
Unfortunately, I think this is an artifact with github. I did not make this change.